doc/stdlib/math.rst

```diff
@@ -114,106 +151,170 @@ Mathematical Operators
 .. _.^:
 .. function:: .^(x, y)
 
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       .^(x, y)
+
    Element-wise exponentiation operator.
 
 .. function:: fma(x, y, z)
 
-   Computes ``x*y+z`` without rounding the intermediate result
-   ``x*y``. On some systems this is significantly more expensive than
-   ``x*y+z``. ``fma`` is used to improve accuracy in certain
-   algorithms. See ``muladd``.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       fma(x, y, z)
+
+   Computes ``x*y+z`` without rounding the intermediate result ``x*y``. On some systems this is significantly more expensive than ``x*y+z``. ``fma`` is used to improve accuracy in certain algorithms. See ``muladd``.
 
 .. function:: muladd(x, y, z)
 
-   Combined multiply-add, computes ``x*y+z`` in an efficient manner.
-   This may on some systems be equivalent to ``x*y+z``, or to
-   ``fma(x,y,z)``. ``muladd`` is used to improve performance. See
-   ``fma``.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       muladd(x, y, z)
+
+   Combined multiply-add, computes ``x*y+z`` in an efficient manner. This may on some systems be equivalent to ``x*y+z``, or to ``fma(x,y,z)``. ``muladd`` is used to improve performance. See ``fma``.
 
 .. function:: div(x, y)
 
-   ::
-              ÷(x, y)
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       div(x, y)
+       ÷(x, y)
 
    The quotient from Euclidean division. Computes ``x/y``, truncated to an integer.
 
 .. function:: fld(x, y)
 
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       fld(x, y)
+
    Largest integer less than or equal to ``x/y``.
 
 .. function:: cld(x, y)
 
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       cld(x, y)
+
    Smallest integer larger than or equal to ``x/y``.
 
 .. function:: mod(x, y)
 
-   Modulus after division, returning in the range [0,``y``), if ``y`` is
-   positive, or (``y``,0] if ``y`` is negative.
+   .. Docstring generated from Julia source
+   .. code-block:: julia
+       mod(x, y)
+
+   Modulus after division, returning in the range [0,``y``), if ``y`` is positive, or (``y``,0\] if ``y`` is negative.
```

Comment:
Looks like some of the escapes that pandoc added are leaking back through to the rst, should be easy to fix